From e746c657b8f853418914360cabf670e84d30707a Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Tue, 30 Oct 2018 17:03:37 +0200
Subject: [PATCH 286/741] net/phy: xgkr: Removed the unnecessary error message
 'No associated device tree node'

commit 30e939196be7bc26721ec53001a1a11651d3eb75 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The error message 'No associated device tree node' issued from callback 'fsl_backplane_match_phy_device' was not necessary and annoying for other usecases without backplane, so it was removed.

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/phy/fsl_backplane.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/phy/fsl_backplane.c b/drivers/net/phy/fsl_backplane.c
index a65ad30c218a..e50ffc1a8716 100644
--- a/drivers/net/phy/fsl_backplane.c
+++ b/drivers/net/phy/fsl_backplane.c
@@ -1656,7 +1656,6 @@ static int fsl_backplane_match_phy_device(struct phy_device *phydev)
 	const int num_ids = ARRAY_SIZE(phydev->c45_ids.device_ids);
 
 	if (!phydev->mdio.dev.of_node) {
-		dev_err(&phydev->mdio.dev, "No associated device tree node\n");
 		return 0;
 	}
 
-- 
2.17.1

